src/components/home/HomePage.scss

```diff
@@ -0,0 +1,351 @@
+#navbar {
+  display: flex;
+  height: 100px;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #6676a0;
+  color: white;
+  justify-content: flex-end;
+  position: relative;
+  padding: 0;
+  overflow-x: hidden;
+
+  .collapseMenuItems {
+    width: 80%;
+    justify-content: flex-end;
+  }
+
+  .navbarItems {
+    width: 50%;
+    padding: 10px 5px 10px 5px;
+
+    > .col {
+      padding-left: 5px;
+      padding-right: 5px;
+      transition: font-size 0.3s ease-in-out;
+    }
+  }
+
+  .navbarLogo {
+    position: absolute;
+    max-width: 100px;
+    left: 100px;
+    cursor: pointer;
+  }
+
+  > div {
+    text-align: center;
+
+    a {
+      text-decoration: none;
+      color: white;
+    }
+  }
+
+  .iconTrio {
+    width: 25%;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 20px;
+    padding: 10px 5px 10px 5px;
+
+    > div {
+      max-width: 35px;
+      margin-right: 15px;
+      min-width: 18px;
+    }
+
+    .fas,
+    .fab {
+      font-size: 22px;
+      cursor: pointer;
+      transition: font-size 0.3s ease-in-out;
+    }
+  }
+
+  @media only screen and (max-width: 980px) {
+    .navbarItems > .col {
+      font-size: 15px;
+    }
+
+    .iconTrio {
+      .fas,
+      .fab {
+        font-size: 20px;
+      }
+    }
+  }
+
+  @media only screen and (max-width: 880px) {
+    .navbarItems > .col {
+      font-size: 14px;
+    }
+
+    .iconTrio {
+      .fas,
+      .fab {
+        font-size: 18px;
+      }
+    }
+  }
+
+  @media only screen and (max-width: 850px) {
+    .navbarItems > .col {
+      font-size: 13px;
+    }
+  }
+
+  @media only screen and (max-width: 770px) {
+    .navbarItems {
+      width: 50%;
+    }
+  }
+
+  .hamburgerContainer {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  #navbar.container-fluid {
```

Comment:
ids

Reply:
actually if you're using sass you can do something like this
http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32

it's better to nest the media query like this
```.navbar.container-fluid {
  // my normal css
  @media 
   // my media css
}
```
because the reader knows exactly what the mobile/non-mobile versions are next to each other